src/Serialization/main.jl

```diff
@@ -435,11 +477,14 @@ include("Upgrades/main.jl")
 # Interacting with IO streams and files
 
 """
-    save(io::IO, obj::Any; metadata::MetaData=nothing)
-    save(filename::String, obj::Any, metadata::MetaData=nothing)
+    save(io::IO, obj::Any; metadata::MetaData=nothing, with_attrs::Bool=false)
+    save(filename::String, obj::Any, metadata::MetaData=nothing, with_attrs::Bool=false)
```

Comment:
This now longer matches the changed default

Reply:
thanks!